NICE.Identity.Management/ClientApp/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@nice-digital/design-system": "^1.2.14-alpha.0",
+    "@nice-digital/design-system": "^1.2.13",
```

Comment:
Could this be changed back to alpha - we need that version so that we can add extra attributes to components, such as the Container one below.